analyzer/src/funTest/assets/projects/synthetic/spdx/project/project.spdx.yml

```diff
@@ -50,6 +50,21 @@ packages:
   name: "openssl"
   versionInfo: "1.1.1g"
   originator: "Organization: OpenSSL Development Team"
+- SPDXID: "SPDXRef-Saxon"
```

Comment:
Commit message:
- In the title, please capitalize after ":", and do not use a dot at the end
- s/project.spdx.ymls/project.spdx.yml's/
- s/spdx.project.yml/project.spdx.yml/

Reply:
"SPDXRef-Saxon" should be "SPDXRef-Packge-Saxon".